### PR TITLE
Header newsletter updates

### PR DESCRIPTION
**Pull request by amanifarooque**
## Summary
- Updated the USWDS Banner component to display current text (https://designsystem.digital.gov/components/banner/)
- Added Newsletters

## PR Checklist
- [x] Test Federalist Preview link for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
- [N/a] Conduct a HemingwayApp review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [x] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [x] Ensure no non-public information is published in the commit messages

### CODE REVIEW

**Comment by amanifarooque** [comment]
Preview URL: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/header-newsletter-updates/about/updates/newsletters/

File: _layouts/post.html [hunk 1]
```diff
@@ -25,6 +25,11 @@ <h1 class="title">{{page.title}}</h1>
           <div class="text-base margin-bottom-2">
             <a href="{{ site.baseurl }}/about/updates/releases">View all release notes</a>
           </div>
+        {% elsif page.category == "newsletters" %}
```
Comment (dawnpm) [inline _layouts/post.html:28]: typo here?

File: _newsletters/2018-06-01-june-2018.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## SEO Pro Tip
+
+### Use `rel=canonical` links if you have multiple URL formats for the same content
+
+Because multiple URLs can serve up the same or similar content, canonical URLs are an important piece to consider to improve your site’s SEO and, ultimately, your visitors’ search experience.
+
+A &quot;canonical link&quot; refers to an HTML `<link>` element, with a `rel=&quot;canonical&quot; attribute`, found in the `<head>` of your webpage. It specifies to search engines your preferred URL for that page.
+
+For example:
+`https://example.gov/topics` vs <br />`http://www.example.gov/topics.html` vs <br />`https://m.example.gov/topics` vs <br />`https://www.example.gov/topics?sort=desc`**<br />**
```
Comment (dawnpm) [inline _newsletters/2018-06-01-june-2018.md:55]: What's up with this bolded <br />?

File: _newsletters/2018-08-01-august-2018.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+In the absence of a main element, search engines will grab the full <body> tag, which would then mean that your header, footer, and navigation text would be indexed along with the “meat” of the page.
+
+Be sure you’ve wrapped the correct content in the main element. One site we indexed recently had done the following, which meant that we only indexed the title of their page:
+
+`<main><h1>This is your page title</h1></main><p>This is the main text of your page`
+
+Hear more about other website-side things you can do to improve indexing of your websites in our recent webinar:
+
+<a href="https://digital.gov/event/2018/05/29/how-search-engines-index-your-websites/?utm_campaign=Search.gov%20Newsletter&amp;utm_source=hs_email&amp;utm_medium=email&amp;_hsenc=p2ANqtz-8cT4jDP1_hA0ORO3uuD8lpypgPvDSIcA8cOb2mhpT0_EqvDYRizCypKeK2Lqnk04zCMfQb">How Search Engines Index Your Websites</a>
```
Comment (dawnpm) [inline _newsletters/2018-08-01-august-2018.md:46]: straggler HS tracking params

File: _newsletters/2019-01-01-january-2019.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+If you’re experiencing an emergency and need immediate assistance during the shutdown, send an email to <a href="mailto:search@gsa.gov">search@gsa.gov</a> including the word “URGENT” in the subject line.
+
+### How to tell searchers about your site's status
+
+Is your website affected by the shutdown? Below are two ways you can post a message on your search results page to tell searchers about your site’s status.
+
+1. Post a search page alert.
+
+    You can add a message to the top of your search results. Searchers will always see your message, regardless of their search term.
```
Comment (dawnpm) [inline _newsletters/2019-01-01-january-2019.md:27]: Is this indent ok? and same question for lines 29, 33, 35, and 37.

File: _newsletters/2019-05-01-may-2019.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+Multiple paths can be combined with a +, for example:
+
+`&sitelimit=www.exampledomain.gov/limittothisfolder+folderarchive.exampledomain.gov`.
+
+### How to add it
+
+Sitelimit parameters can be added via an input in the search box
+
+```<input type="hidden" name="sitelimit" id="sitelimit" value="www.exampledomain.gov/limittothisfolder">```
```
Comment (dawnpm) [inline _newsletters/2019-05-01-may-2019.md:37]: do we need to change these code block backtick characters to their own lines? looks like the rest of the code is confused, and still blue.

File: _newsletters/2019-05-01-may-2019.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+### Make the Switch!
+
+Here are some helpful steps to customize your search results page and match your site’s brand before **June 28, 2018:**
+
+**Create a new site**
+
+Use the Add Site button at the top of the Admin Center to create a new site. Here is where you can play around with the look and feel before going live.
+
+<a href="https://search.gov/manual/brand.html?utm_source=hs_email&amp;utm_medium=email&amp;_hsenc=p2ANqtz-8asXLD8kW1r6huIYx7xdkq4QtNA47llNtSUy9jkEVwU5dRiMUYsSnNV0lCI1aMs138m2T3">**Define your color scheme on the Fonts &amp; Colors page**
```
Comment (dawnpm) [inline _newsletters/2019-05-01-may-2019.md:71]: where does this anchor tag end?

File: _newsletters/2019-08-01-august-2019.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## It's That Time of the Year
+
+### Site Redesign and Relaunch
+
+Read our <a href="https://search.gov/manual/redesign.html">Checklist for a Successful Site Redesign</a> (newly updated with additional information!).
+
+See the <a href="https://search.gov/manual/metadata.html">metadata and tags you should include in your new page templates</a>.
+
+Be sure to publish an <a href="https://search.gov/manual/sitemaps.html">xml sitemap</a><span style="font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif; font-size: 16px; background-color: transparent;"> and let us know about it, so we can tell our system to start indexing your content as soon as it’s public.
```
Comment (dawnpm) [inline _newsletters/2019-08-01-august-2019.md:36]: straggler HS inline style tag

File: _newsletters/2019-11-01-november-2019.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+We're now using Login.gov to add MFA to our service. We also marked our Drupal module as unsupported, which it's been for a while.
+
+## Multi-Factor Authentication
+
+### Integration with Login.gov is Complete
+
+We are officially integrated with Login.gov as our authentication service to increase security in our system. You will now need to provide secondary identification along with your username and password to successfully access your Search.gov accounts.
+
+If you haven't already, **you need to set up an account with Login.gov,** and no worries, the rest of your Search.gov experience will remain the same. If you don't already have an account, you can set one up <a href="https://search.usa.gov/login?utm_campaign=Search.gov%20Newsletter&amp;utm_source=hs_email&amp;utm_medium=email&amp;_hsenc=p2ANqtz-_LkFD_kPta_wIsJy28mibrFgKkjTfZCS4vJXdrD8Hi3wfx3MBxq7UrsucbNvIsohR16Q8l">while logging in to our service</a>. Please be sure that the email address you use with us is listed in your Login.gov account.
```
Comment (dawnpm) [inline _newsletters/2019-11-01-november-2019.md:21]: straggler HS tracking codes

File: _newsletters/2019-11-01-november-2019.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+If you currently use a group email account, you will need to add individual people’s email addresses as users, and use those individual accounts to access the system, rather than the group address.
+
+If you have any questions about this or anything else, please <a href="mailto:search@gsa.gov">reach out to us</a>.
+
+## Attention Drupal Sites
+
+### “usasearch” Not Supported (But All is Well)
+
+As <a href="https://search.gov/blog/six-months-in.html?utm_campaign=Search.gov%20Newsletter&amp;utm_source=hs_email&amp;utm_medium=email&amp;_hsenc=p2ANqtz-_LkFD_kPta_wIsJy28mibrFgKkjTfZCS4vJXdrD8Hi3wfx3MBxq7UrsucbNvIsohR16Q8l">announced</a> in 2018, we stopped supporting our Drupal module as it wasn’t a feasible solution for most customers. At that time, we moved to <a href="https://search.gov/manual/indexing-with-searchgov.html?utm_campaign=Search.gov%20Newsletter&amp;utm_source=hs_email&amp;utm_medium=email&amp;_hsenc=p2ANqtz-_LkFD_kPta_wIsJy28mibrFgKkjTfZCS4vJXdrD8Hi3wfx3MBxq7UrsucbNvIsohR16Q8l">indexing</a> primarily from XML sitemaps because they're a core component of SEO and will support search coverage in our system, as well as out in the commercial search engines.
```
Comment (dawnpm) [inline _newsletters/2019-11-01-november-2019.md:31]: straggler HS tracking codes

File: _newsletters/2020-02-01-february-2020.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+**Make sure your website loads fast:** one of the relatively easy things you can do for your multilingual SEO (and SEO in general) is to make your website fast as it plays a role as a ranking factor for search engines.
+
+[**Read Google's tips for multilingual SEO**](https://support.google.com/webmasters/answer/182192?hl=en)
+
+## In Case You Missed It
+
+### The searches that shaped 2019 and our System Highlights
+
+[![2019_annual_review_small](https://cdn2.hubspot.net/hub/1962994/hubfs/2019_annual_review_small.png?upscale=true&width=394&upscale=true&name=2019_annual_review_small.png)](https://search.gov/blog/2019-annual-review.html)
```
Comment (dawnpm) [inline _newsletters/2020-02-01-february-2020.md:64]: we have native thumbnails we can use here, right?

File: _layouts/post.html (hunk 1 above)
Comment (amanifarooque) [reply _layouts/post.html:28]: elsif is correct - https://shopify.github.io/liquid/tags/control-flow/

File: _newsletters/2020-05-01-may-2020.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+![Screen Shot 2020-06-01 at 2.27.35 PM](https://cdn2.hubspot.net/hub/1962994/hubfs/Screen%20Shot%202020-06-01%20at%202.27.35%20PM.png?upscale=true&width=800&upscale=true&name=Screen%20Shot%202020-06-01%20at%202.27.35%20PM.png) Some users who work with multiple GSA programs may not receive the Search.gov survey. Even if you do not receive the survey, we would still love to hear from you. Please reach out to us by email, and we can send you the questions separately. Thank you in advance! ## System Update ### Upgrades Completed Ahead of Finalizing ATO Renewal We’ve been focused on upgrading our software for [Search.gov](http://search.gov/)'s back end, in anticipation of completing our Authority to Operate (ATO) renewal process. Our most recent upgrades include Elasticsearch, Ubuntu, Rails, and other smaller components to ensure reliability of continued service. 🎉
```
Comment (dawnpm) [inline _newsletters/2020-05-01-may-2020.md:17]: do we have a native version of this image?

File: _newsletters/2019-01-01-january-2019.md (hunk 4 above)
Comment (amanifarooque) [reply _newsletters/2019-01-01-january-2019.md:27]: yep : https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/header-newsletter-updates/about/updates/newsletters/january-2019.html

File: _newsletters/2018-08-01-august-2018.md (hunk 3 above)
Comment (amanifarooque) [reply _newsletters/2018-08-01-august-2018.md:46]: will remove

File: _newsletters/2019-05-01-may-2019.md (hunk 6 above)
Comment (amanifarooque) [reply _newsletters/2019-05-01-may-2019.md:71]: will fix, but seems to be displaying right: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/header-newsletter-updates/about/updates/newsletters/may-2019.html

File: _newsletters/2021-01-01-january-2021.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+In Drupal 7, file attachments needed to be added to the XML sitemap manually, one by one. This is still the case in later versions of Drupal if you’re using the Simple XML Sitemap module. For many teams, this is not a sustainable way to ensure PDFs appear in search results.
+
+We recommend that all Drupal sites use the [XML Sitemap module](https://www.drupal.org/project/xmlsitemap), if possible: it has a similar name, but different features. This module allows you to include file attachments globally by type, with a simple check box within the module settings, via the Sitemap Entities tab.
+
+![Screen Shot 2021-01-29 at 10.46.21 AM](https://cdn2.hubspot.net/hub/1962994/hubfs/Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png?upscale=true&width=1000&upscale=true&name=Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png)
```
Comment (dawnpm) [inline _newsletters/2021-01-01-january-2021.md:25]: do we have a native version of this?

File: _newsletters/2021-03-01-march-2021.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+sidenav: about_us
+---
+
+## Summary
+We're pleased to release our 2020 Year In Review. We also did a big cleanup of outdated typeahead suggestions.
+
+## 2020 Year in-Review
+### The searches that shaped 2020 and our system highlights
+
+[![2020-Year-In-Review-Cover-Full](https://cdn2.hubspot.net/hub/1962994/hubfs/2020-Year-In-Review-Cover-Full.png?upscale=true&width=700&upscale=true&name=2020-Year-In-Review-Cover-Full.png)]({{ site.baseurl }}/about/updates/year-in-review/2020-year-in-review.html)
```
Comment (dawnpm) [inline _newsletters/2021-03-01-march-2021.md:17]: let's use our native version of this

File: _newsletters/2020-05-01-may-2020.md (hunk 11 above)
Comment (amanifarooque) [reply _newsletters/2020-05-01-may-2020.md:17]: I dont' think so, this is the survey email so wouldn't make sense to put on our site.

File: _newsletters/2021-07-01-july-2021.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+In the Simple XML Sitemap module, file attachments need to be added to the XML sitemap manually, one by one. For many teams, this is not a sustainable way to ensure PDFs appear in search results.
+
+We recommend that all Drupal sites use the [XML Sitemap module](https://www.drupal.org/project/xmlsitemap), if possible: it has a similar name, but different features. This module allows you to include file attachments globally by type, with a simple check box within the module settings, via the Sitemap Entities tab.
+
+![Screen Shot 2021-01-29 at 10.46.21 AM](https://cdn2.hubspot.net/hub/1962994/hubfs/Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png?upscale=true&width=1120&upscale=true&name=Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png)
```
Comment (dawnpm) [inline _newsletters/2021-07-01-july-2021.md:30]: Do we have a native version of this?

File: _newsletters/2021-08-01-august-2021.md [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+sidenav: about_us
+---
+
+## Summary
+Do you have a transition coming in your web team? Follow these steps to ensure the new team will be able to access and manage your search.
+
+##  Gaining Access to An Existing Site
+### How to Access Analytics
+
+If you need to gain access to the analytics data for an existing search site account, ask a team member at your agency to [add you as a user](https://search.gov/manual/users.html?utm_campaign=Search.gov%20Newsletter&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz--FaF_rXTvCis-9LSjmV5KNI3Utxhwnkp4z9_Y9Ae74R6B85Na9bi_w_U2ijostFvCEL_v7).
```
Comment (dawnpm) [inline _newsletters/2021-08-01-august-2021.md:17]: straggler HS tracking codes

File: _newsletters/2022-04-01-april-2022.md [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+### Search and Website Redesign: Giving Us a Heads-up Is Appreciated!
+
+If you are planning a website redesign, whether it’s an upgrade or adopting a new content management system, we encourage you to think about its impact on your search results. A good way to start factoring search into your redesign project is by letting us know ahead of time of your plans and if possible, the timeline you have set for them. Being aware of your redesign plans allows us to work together with you and prepare for this transition. To get started, we suggest you read our [checklist for a successful website redesign](https://search.gov/indexing/redesign.html).
+
+## Search.gov Website Redesigned
+### Let Us Know What You Think
+
+The new Search.gov website has been live since January 2022. We redesigned and rebuilt it to better support federal web teams, both existing customers and those interested in learning more about our service. 
+
+Now we ask you to please share your feedback by completing this brief [survey](https://touchpoints.app.cloud.gov/touchpoints/47b65524/submit?_hsmi=2&utm_content=2). As usual, your input is very important to us. Thank you for your time!
```
Comment (dawnpm) [inline _newsletters/2022-04-01-april-2022.md:38]: are these straggler HS tracking codes?

File: _newsletters/2022-05-01-may-2022.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+### Let Us Know What You Need But We Don’t Have Yet
+
+We are working very hard to deliver the best possible search experience for you and your site’s visitors. Our goal is to improve our service by growing its capabilities based on our customers’ search needs. For this reason we ask you to please share with us those features that you think will make your search journey better and easier for you and your visitors. Your feedback is key to helping us make Search.gov stronger. Please use [this form](https://touchpoints.app.cloud.gov/touchpoints/e3fc260e/submit) to share your features requests. Thank you!
+
+## Search.gov Website Redesigned
+### Let Us Know What You Think
+
+The new Search.gov website has been live since January 2022. We redesigned and rebuilt it to better support federal web teams, both existing customers and those interested in learning more about our service. 
+
+Now, almost 120 days after it was relaunched, we ask you to please share your feedback by completing this brief [survey](https://touchpoints.app.cloud.gov/touchpoints/47b65524/submit?_hsmi=2&utm_content=2). As usual, your input is very important to us. Thank you for your time!
```
Comment (dawnpm) [inline _newsletters/2022-05-01-may-2022.md:40]: are these straggler HS tracking codes?

**Review by dawnpm — CHANGES_REQUESTED** [review]
This is great, thanks for bringing this over the finish line.

I have several questions/comments, which it looks like you're already addressing, so we'll be in good shape there.

Idea for later - add or just move summary to description, and include in the Newsletters landing page, under the link to the date.

File: _newsletters/2020-05-01-may-2020.md (hunk 11 above)
Comment (dawnpm) [reply _newsletters/2020-05-01-may-2020.md:17]: The image will break in a few months, though - maybe just remove it entirely?

File: _newsletters/2021-01-01-january-2021.md (hunk 12 above)
Comment (dawnpm) [reply _newsletters/2021-01-01-january-2021.md:25]: don't want the image reference to break. so we should either create a version of the file on our site, or remove from the newsletter.

File: _newsletters/2021-07-01-july-2021.md (hunk 14 above)
Comment (dawnpm) [reply _newsletters/2021-07-01-july-2021.md:30]: don't want the image reference to break. so we should either create a version of the file on our site, or remove from the newsletter.

File: _newsletters/2020-05-01-may-2020.md (hunk 11 above)
Comment (amanifarooque) [reply _newsletters/2020-05-01-may-2020.md:17]: removing

File: _newsletters/2018-06-01-june-2018.md (hunk 2 above)
Comment (amanifarooque) [reply _newsletters/2018-06-01-june-2018.md]: Removed

File: _newsletters/2021-01-01-january-2021.md (hunk 12 above)
Comment (amanifarooque) [reply _newsletters/2021-01-01-january-2021.md]: added the image file

**Comment by amanifarooque** [comment]
@dawnpm I believe I've addressed all comments, let me know when good to merge!

**Review by dawnpm — APPROVED** [review]
looks good, thanks!

**Pull request merged**